src/memberKeyImportToken.ts

```diff
@@ -50,13 +50,17 @@ export async function processMemberKeyImportToken(
     keyImportData.publicKeyImportToken,
   );
   if (!memberKeyImportToken) {
+    options.logger.info(
+      { id: keyImportData.publicKeyImportToken },
```

Comment:
We refer to this as "token", not "id", so I think we should be consistent. "Id" is too ambiguous -- is the the member id?